src/model/mosaic/MosaicFlags.ts

```diff
@@ -80,4 +91,27 @@ export class MosaicFlags {
             flags: this.getValue(),
         };
     }
+
+    /**
+     * It "adds up" individual flags into a bit wise number flag.
+     *
+     * @param supplyMutable - if the supply is mutable. First flag.
+     * @param transferable - if the balance can be transferred. Second flag.
+     * @param restrictable - if the transaction can be restricted. Third flag.
+     * @param revokable - if the balance can be revoked. Fourth flag.
+     * @private
+     */
+    private static toFlag({
```

Comment:
`Private static`?

Reply:
Yes, helper private method. Users still use the old constructor or getValue() methods.

Reply:
I don't think private modifier works the same in TS like Java. private is probably not needed as static is always public